src/main/java/school/lemon/changerequest/java/vehicles/VehicleFactory.java

```diff
@@ -7,41 +7,136 @@ private VehicleFactory() {
 
     /**
      * Creates car
+     *
      * @param manufacturedYear manufactured year
-     * @param make manufacturer
-     * @param model of car
-     * @param horsepower of car
+     * @param make             manufacturer
+     * @param model            of car
+     * @param horsepower       of car
      * @return instance of car
      */
     public static ICar createCar(int manufacturedYear, String make, String model, int horsepower) {
-        // TODO: please implement me
-        return null;
+        return new Car(manufacturedYear, make, model, horsepower) {
```

Comment:
there is no need for such duplication. Please just return new instance of non-abstract class here and below.